packages/shared-metrics/src/memory.js

```diff
@@ -9,7 +9,7 @@ exports.payloadPrefix = 'memory';
 // @ts-ignore
 exports.currentPayload = {};
 
-/** @type {NodeJS.Timer} */
+/** @type {NodeJS.Timeout } */
```

Comment:
@kirrg001 Please take a look at these modifications.

Reply:
I guess this looks good when the build is green :)